docs/topics/kmm/kmm-getting-started.md

```diff
@@ -1,22 +1,54 @@
-[//]: # (title: Getting started)
+[//]: # (title: Get started with Kotlin Multiplatform Mobile)
+[//]: # (description: cross platform mobile development, android vs ios development)
 
-## Get familiar with KMM
+> Kotlin Multiplatform Mobile is in [Alpha](components-stability.md). Language features and tooling may change in future
+> Kotlin versions.
+> 
+> Beta is expected in spring 2022. See [KMM Beta Roadmap](https://youtu.be/LE-Dy9XCLxk) video
+> to learn about upcoming features.
+>
+{type="note"}
+
+Kotlin Multiplatform Mobile (KMM) is an SDK designed to simplify development of cross-platform mobile applications.
+You can share common code between iOS and Android apps and write platform-specific code only where it's necessary.
+For example, to implement a native UI or when working with platform-specific APIs.
 
 Watch the introductory [video](https://www.youtube.com/watch?v=mdN6P6RI__k), in which Kotlin Developer Advocate Ekaterina 
```

Comment:
Kate is not a developer advocate anymore. Please contact her to agree on her title.